sdk/communication/azure-communication-chat/src/main/java/com/azure/communication/chat/implementation/models/SendTypingNotificationRequest.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.communication.chat.implementation.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/** Request payload for typing notifications. */
+@Fluent
+public final class SendTypingNotificationRequest {
```

Comment:
We already have this class as TypingNotificationOptions

Reply:
This transformation from the Swagger/Readme should've renamed it, if it doesn't anymore something might've changed:

directive:
- from: swagger-document
  where: $.definitions
  transform: >
    if (!$.TypingNotificationOptions) {
      $.TypingNotificationOptions = $.SendTypingNotificationRequest;
      delete $.TypingNotificationRequest;
    }
- from: swagger-document
  where: $["paths"]["/chat/threads/{chatThreadId}/typing"].post.parameters[2]
  transform: >
    if ($.schema && $.schema.$ref && $.schema.$ref.endsWith("SendTypingNotificationRequest")) {
        const path = $.schema.$ref.replace(/[#].*$/, "#/definitions/TypingNotificationOptions");
        $.schema = { "$ref": path };
    }

Reply:
`delete $.TypingNotificationRequest;` 
The delete param name is incorrect here. Fixed it in the new iteration.